src/components/Variables/VariableInternalComment.tsx

```diff
@@ -59,7 +60,7 @@ function VariableInternalComment({ variable, update, editing, sx }: VariableInte
             onChange={(value) => setInternalComment(value as string)}
             preserveNewlines
             type='textarea'
-            value={internalComment || strings.NO_COMMENTS_ADDED}
+            value={variableValue?.internalComment || strings.NO_COMMENTS_ADDED}
```

Comment:
The field value shouldn't change as the user is typing in the dialog, it shouldn't change until the user presses Save